README.md

```diff
@@ -1,18 +1,18 @@
-# DeltaV System Spec
+# Suborbital System Spec
 
-This repository contains the specification and canonical Go implementions of the core DeltaV system. DeltaV uses a control plane / data plane architecture which allows the system as a whole to be distributed in various ways. There are five main components:
+This repository contains the specification and canonical Go implementions of the core Suborbital system. Suborbital uses a control plane (SE2) / data plane (E2Core) architecture which allows the system as a whole to be distributed in various ways. There are five main components:
 
-## AppSource
-The main interface that allows a DeltaV data plane instance to learn its state from a DeltaV control plane server.
+## System
+The main interface that allows a data plane instance to learn its state from a control plane server.
 
 ## Tenant
-Also known as the `tenant.yaml` file or 'tenant config', this defines the declarative schema for describing a DeltaV tenant
+Also known as the `tenant.json` file or 'tenant config', this defines the declarative schema for describing a Suborbital tenant
```

Comment:
It looks like we're still using `yaml.Unmarshal` below for the tenant config.

Reply:
The default is JSON, but I added secondary 'UnmarshalYaml'